src/main/resources/assets/computercraft/lua/rom/apis/window

```diff
@@ -270,19 +270,19 @@ function create( parent, nX, nY, nWidth, nHeight, bStartVisible )
     end
 
     local function setTextColor( color )
-        nTextColor = color
-        if bVisible then
-            updateCursorColor()
+        if type(color) == "number" then
+	    nTextColor = color
+            if bVisible then
+               updateCursorColor()
+            end
+        else
+           error("Color must be a number")
```

Comment:
Would probably be better to use `error("Expected number", 2)` here, as that is what the `term` API uses.